backend/ops_api/tests/ops/can/test_budget_line_item.py

```diff
@@ -106,7 +106,7 @@ def test_post_budget_line_items(loaded_db, auth_client, test_can):
         "proc_shop_fee_percentage": 1.23,
         "services_component_id": 1,
     }
-    response = auth_client.post("/api/v1/budget-line-items/", json=data)
+    response = division_director_auth_client.post("/api/v1/budget-line-items/", json=data)
```

Comment:
I don't think we want to limit operations on BLIs, e.g. creating a new BLI, to only the division director.  These tests should still work with the regular unit test user, i.e. the `auth_client` user.

Reply:
Good call. I over-corrected originally when switching the auth client. All the posts are using the basic auth_client now and only the patch requests use the division director auth client.

Reply:
Also the reality is that the tests in test_budget_line_items are only updated because when I changed the change requests these tests were breaking. If I didn't include these fixes my backend tests would not pass.

Reply:
Sorry my fixes for test_budget_line were not needed and a knockoff error caused by my other configuration problems.